src/Symfony/Component/Security/Guard/AbstractGuardAuthenticator.php

```diff
@@ -30,7 +30,7 @@ abstract class AbstractGuardAuthenticator implements AuthenticatorInterface
      *
      * @return PostAuthenticationGuardToken
      */
-    public function createAuthenticatedToken(UserInterface $user, $providerKey)
+    public function createAuthenticatedToken(UserInterface $user, string $providerKey)
```

Comment:
Please also remove both `@param` annotations because they‘re redundant now.

Reply:
Given I removed the `@param UserInterface $user` annotation for 3.4 here https://github.com/symfony/symfony/pull/32257

I just removed the `@param string $provider` annotations in this PR.